src/extension/options-page/DashboardComponents/PersonaCard.tsx

```diff
@@ -87,6 +87,10 @@ export default function PersonaCard({ persona }: Props) {
 
     const [deletePersona, setDeletePersona] = useState(false)
     const confirmDeletePersona = () => {
+        // snackbar will cover button
+        if (webpackEnv.target === 'E2E') {
```

Comment:
🤔maybe wait for the snackbar to disappear or click the button manually instead of this way?